bigquery_etl/dependency.py

```diff
@@ -13,44 +14,17 @@
 
 stable_views = None
 
-try:
-    import jnius_config  # noqa: E402
-
-    if not jnius_config.vm_running:
-        # this has to run before jnius is imported the first time
-        target = Path(__file__).parent.parent / "target"
-        for path in target.glob("*.jar"):
-            jnius_config.add_classpath(path.resolve().as_posix())
-except ImportError:
-    # ignore so this module can be imported safely without java installed
-    pass
-
 
 def extract_table_references(sql: str) -> List[str]:
     """Return a list of tables referenced in the given SQL."""
-    # import jnius here so this module can be imported safely without java installed
-    import jnius  # noqa: E402
-
-    try:
-        ZetaSqlHelper = jnius.autoclass("com.mozilla.telemetry.ZetaSqlHelper")
-    except jnius.JavaException:
-        # replace jnius.JavaException because it's not available outside this function
-        raise ImportError(
-            "failed to import java class via jni, please build java dependencies "
-            "with: mvn package"
-        )
-    try:
-        result = ZetaSqlHelper.extractTableNamesFromStatement(sql)
-    except jnius.JavaException:
-        # Only use extractTableNamesFromScript when extractTableNamesFromStatement
-        # fails, because for scripts zetasql incorrectly includes CTE references from
-        # subquery expressions
-        try:
-            result = ZetaSqlHelper.extractTableNamesFromScript(sql)
-        except jnius.JavaException as e:
-            # replace jnius.JavaException because it's not available outside this function
-            raise ValueError(*e.args)
-    return [".".join(table.toArray()) for table in result.toArray()]
+
+    parsed = sqlglot.parse_one(sql, read="bigquery")
```

Comment:
it might be more robust to do 

```python

from sqlglot import exp
from sqlglot.optimizer.scope import traverse_scope

def find_tables(query: exp.Expression) -> t.Set[str]:
    """Find all tables referenced in a query.

    Args:
        query: The expression to find tables for.

    Returns:
        A Set of all the table names.
    """
    return {
        exp.table_name(table)
        for scope in traverse_scope(query)
        for table in scope.tables
        if isinstance(table.this, exp.Identifier)
        and exp.table_name(table) not in scope.cte_sources
    }
```